client/extensions/woocommerce/state/sites/shipping-methods/test/selectors.js

```diff
@@ -116,4 +121,56 @@ describe( 'selectors', () => {
 			expect( areShippingMethodsLoading( stateLoading ) ).to.be.true;
 		} );
 	} );
+
+	describe( 'getShippingMethodNameMap', () => {
+		it( 'should return unknown if the methods are loading', () => {
```

Comment:
This test is not quite returning unknown. It just returns the ID back. Should it return something else like undefined, or should the test description just be updated?